### PR TITLE
[DE] Add `Approach_Rate`

### PR DESCRIPTION
**Pull request by Warpezz**
(This is my first time using Github so apologies for any mistakes I've made in the whole Pull Req process)

This is a draft for the German translation of the approach rate wiki page.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by The-Last-Cookie** [comment]
Hello! Is this ready for review? There is a TODO at the end of the file.

**Comment by Warpezz** [comment]
> Hello! Is this ready for review? There is a TODO at the end of the file.

yes i think it should be now

File: wiki/Beatmap/Approach_rate/de.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- AR < 5: `Verzug = 1200ms + 600ms * (5 - AR) / 5` <!-- verzug? ich weiß nich wie man 'preempt' übersetzen soll -->
+- AR = 5: `Verzug = 1200ms`
+- AR > 5: `Verzug = 1200ms - 750ms * (AR - 5) / 5`
+
+Die Zeitspanne benötigt für das Objekt komplett zu erscheinen hängt auch von dem AR ab:
+
+- AR < 5: `erscheinen = 800ms + 400ms * (5 - AR) / 5`
+- AR = 5: `erscheinen = 800ms`
+- AR > 5: `erscheinen = 800ms - 500ms * (AR - 5) / 5`
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:39]:
```suggestion
- AR > 5: `Zeitspanne zum Einblenden = 800 ms - 500 ms * (AR - 5) / 5`
```

File: wiki/Beatmap/Approach_rate/de.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Mod Beeinflussungen
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:41]:
```suggestion
## Mod-Effekte
```
or
```suggestion
## Effekte durch Mods
```

File: wiki/Beatmap/Approach_rate/de.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Es gibt vier Mods, die die AR beeinflussen, wenn sie aktiviert sind:
+
+- [Easy](/wiki/Game_modifier/Easy): Halbiert die AR.
+- [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multipliziert die AR mit 1.4 (maximal 10).
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:46]:
```suggestion
- [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multipliziert die AR mit 1,4 (maximal 10).
```

File: wiki/Beatmap/Approach_rate/de.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- [Double Time](/wiki/Game_modifier/Double_Time): Die AR wird nicht beeinflusst, aber wegen der 50% Erhöhung der Geschwindigkeit bleiben Objekte 33% weniger lange sichtbar.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:47]:
```suggestion
- [Double Time](/wiki/Game_modifier/Double_Time): Die AR wird nicht beeinflusst, aber wegen der Geschwindigkeitserhöhung um 50 % bleiben Hit-Objekte 33 % weniger lange sichtbar.
```
Only a minor nitpick: [The percent sign is written with a space in this case in German](https://www.typolexikon.de/prozentzeichen/)

File: wiki/Beatmap/Approach_rate/de.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- [Half Time](/wiki/Game_modifier/Half_Time): Die AR ist nicht beeinflusst, aber wegen des 25% Rückgangs der Geschwindigkeit bleiben die Objekte für eine um 33% längere Zeit sichtbar.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:48]:
```suggestion
- [Half Time](/wiki/Game_modifier/Half_Time): Die AR ist nicht beeinflusst, aber wegen der Geschwindigkeitsreduzierung um 25 % bleiben Hit-Objekte für eine um 33 % längere Zeit sichtbar.
```

File: wiki/Beatmap/Approach_rate/de.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Obwohl die "Half Time" und "Double Time" Mods keinen Einfluss auf das AR haben, bewirkt die Änderung der Geschwindigkeit einen offensichtlichen Unterschied in der AR. HT/DT ARs sind häufig bezeichnet gemäß der wahrgenommene Größe. Zum Beispiel, "AR8+DT" darf auch als "AR9.6" geschrieben werden.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:50]:
```suggestion
Obwohl die Mods "Half Time" und "Double Time" keinen Einfluss auf die AR haben, bewirkt die Änderung der Geschwindigkeit einen scheinbaren Unterschied im AR-Wert. ARs mit HT/DT werden häufig gemäß ihrer wahrgenommenen Werte bezeichnet. Zum Beispiel darf "AR8+DT" auch als "AR9.6" geschrieben werden.
```
Is a bit easier to read in my opinion.

File: wiki/Beatmap/Approach_rate/de.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## osu!taiko und osu!mania
+
+Approach-Rate ist sichtbar wenn die Beatmap-Informationen von [osu!taiko](/wiki/Game_mode/osu!taiko) oder [osu!mania](/wiki/Game_mode/osu!mania) angeizeigt werden, aber es hat keinen Einfluss auf das Gameplay. Das Scrollspeed (Bildlaufgeschwindigkeit) in jedem Spielmodus wird durch andere Faktoren bestimmt.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:54]:
```suggestion
Die AR ist sichtbar, wenn Beatmap-Informationen von [osu!taiko](/wiki/Game_mode/osu!taiko) oder [osu!mania](/wiki/Game_mode/osu!mania) angezeigt werden, aber es hat keinen Einfluss auf das Gameplay. Die Scrollgeschwindigkeit (Bildlaufgeschwindigkeit) in jedem Spielmodus wird durch andere Faktoren bestimmt.
```

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
In case you are learning German, I wanted to let you know that I really appreciate your effort. More so with wiki articles. Translation is sometimes challenging, even for me.

One thing that I noticed is that you used your `master`. Please use a feature branch for that. For information about GitHub, see [best practices § making edits](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)

File: wiki/Beatmap/Approach_rate/de.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multipliziert die AR mit 1,4 (maximal 10).
+- [Double Time](/wiki/Game_modifier/Double_Time): Die AR wird nicht beeinflusst, aber wegen der Geschwindigkeitserhöhung um 50 % bleiben Hit-Objekte 33 % weniger lange sichtbar.
+- [Half Time](/wiki/Game_modifier/Half_Time): Die AR ist nicht beeinflusst, aber wegen der Geschwindigkeitsreduzierung um 25 % bleiben Hit-Objekte für eine um 33 % längere Zeit sichtbar.
+
+Obwohl die Mods "Half Time" und "Double Time" keinen Einfluss auf die AR haben, bewirkt die Änderung der Geschwindigkeit einen scheinbaren Unterschied im AR-Wert. ARs mit HT/DT werden häufig gemäß ihrer wahrgenommenen Werte bezeichnet. Zum Beispiel darf "AR8+DT" auch als "AR9.6" geschrieben werden.
+
+## osu!taiko und osu!mania
+
+Die AR ist sichtbar, wenn Beatmap-Informationen von [osu!taiko](/wiki/Game_mode/osu!taiko) oder [osu!mania](/wiki/Game_mode/osu!mania) angezeigt werden, aber es hat keinen Einfluss auf das Gameplay. Die Scrollgeschwindigkeit (Bildlaufgeschwindigkeit) in jedem Spielmodus wird durch andere Faktoren bestimmt.
+
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Approach_rate/de.md:55]:
```suggestion
```

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
Every article in the wiki needs to have *one* blank line after the last paragraph. In Visual Studio Code for example, this looks like this (you can choose any editor you like though):

line

Otherwise, this article is looking good!

**Comment by Warpezz** [comment]
Thank you lastcookie my saviour

**Review by The-Last-Cookie — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**